docs/contributing/managing-pull-requests.md

```diff
@@ -0,0 +1,178 @@
+- Blog posts are similar to documentation in that they're long form writing, but also want to ensure that blog posts on [gatsbyjs.org](https://www.gatsbyjs.org) aren't purely promotional, spammy or inappropriate.
+
+## Automated Checks
+
+Our repository on [GitHub](https://github.com/gatsbyjs/gatsby) has several automated CI checks that are run automatically for all PRs. These include tests, linting and even preview builds [gatsbyjs.org](https://www.gatsbyjs.org).
+
+We want all of these checks to pass. While it's okay to review a work in progress PR with some failed checks, a PR is only ready to ship when everything is green.
+
+Let's go over some common failure cases in our cases and how to approach fixing them:
+
+### Linting
+
+We lint all code and documentation pretty aggressively (consistency is good) and while that is great, you might find some PRs failing the linting check.
+
+An easy fix for this is to run:
+
+> 💡 npm run format
+
+This will format all code and documentation and should get rid of those pesky linting issues!
+
+## Other Checks
+
+### Testing Locally
+
+While we have many _many_ tests in our repository (that are run automatically on every commit), there can be times when there exists an edge case (or five) that isn't covered by these.
+
+In situations like this, testing the change locally can be very valuable.
+
+> 💡 In case this is the first time you're doing this, you might have to [set up your development environment](https://www.gatsbyjs.org/contributing/setting-up-your-local-dev-environment).
+
+Testing out unpublished packages locally can be tricky. We have just the tool to make that easy for you.
+
+Say hello to your new best friend, `gatsby-dev-cli`.
+
+#### gatsby-dev-cli
+
+`gatsby-dev-cli` is a command-line tool for local Gatsby development. When making changes in gatsby packages, this helps copy changes in the packages to a Gatsby site that you can test your changes on.
+
+Learn more about `gatsby-dev-cli` [here](https://github.com/gatsbyjs/gatsby/tree/master/packages/gatsby-dev-cli)!
+
+### Commit and PR Title
+
+It's good to have descriptive commit messages so that other folks can make sense of what your commit is doing. We like [conventional commits](https://www.conventionalcommits.org/en/v1.0.0-beta.3).
+
+However, PRs get squashed when merged into our repository so individual commit messages are not as important as PR titles.
+
+Let's look at some examples of good and bad PR titles:
+
+#### Good PR Titles ✅
+
+- chore(docs): Fix links in contributing page
+- feat(gatsby): Add support for per page manifests
+- fix(gatsby-plugin-sharp): Ensure images exist before attempting conversion
+
+#### Bad PR Titles ❌
+
+- new tests
+- add support for my new cms
+- fix bug in gatsby
+
+## Giving Feedback
+
+- Be _nice_. We're stronger everyday because of our community so empathy is important and we want our contributors to feel welcome.
```

Comment:
```suggestion
- Be _kind_. We're stronger every day because of our community, so compassion is important. We want all contributors to feel welcome.
```